prebid-mobile/prebid-mobile-rendering.md

```diff
@@ -0,0 +1,110 @@
+
+In the scope of the Open Beta release, you can use the Core and Rendering modules alongside each other. The current integration will work as is without changes but if you want to add or switch to the rendering integration you will have to make changes in the codebase.  
+
+The rendering module consist of three libraries:
+
+- **PrebidSDKRendering**, which introduces the classes for integration into UI and rendering engine
+- [OPTIONAL] **PrebidSDKGAMEventHandlers**, which wraps GAM classes in order to support GAM integration. 
+- [OPTIONAL] **PrebidSDKMoPubAdapters**, which wraps MoPub Adapters for Prebid Rendering module
+
+Using rendering module you can integrate next ad formats:
+
+- **Dispaly** - Banner/Interstitial
+- **Outsrteam Video** - Banner/Interstitial/Rewarded
+- **Native** - Disaply / Video
+
+Intefration steps are convinient and described in the respective docs for [iOS](TBD) and [Android](TBD). 
+
+## Motivation and Benefits 
+
+Rendering serves the general Prebid's aim - to be a trusted, transparent, and configurable monetization solution and gives a lot of benefits to both sides publishers and demand partners. 
+
+In comparising with no-rendering approach the new module gives next benefits:
+
+- Monitizaion without Ad Server. Publishers can monitize the apps through pure In-App bidding without additional ad requests.
+- Faster ad delivery. As soon as bid is recieved it can be displayed. It is applayed to both kind of integration with ad server  or without it.
+- Litle infrastructure. Since the rendering module doesn't need Prebid Cache it can be ommited. 
+- Less discrepancy, better analytics. Rendering module is able to track all needed events in the proper time according to the industry standard or demand side custom requrements. 
+- Full support of SKAdNetworks and similar frameworks 
+- MRAID 3.0 support
+- Better measurement. Ability to use any measurement provider. SDK implements the support of Open Measurement, the certification is scheduled to the upcoming releases.  
+- Open for improvements and adding exclusive features 
+- Support of Prebid Server distinct features like PG, omni-channel, floors, etc.  
+
+## How It Works
+
+The following diagrams show how the Rendering Module works. So far it supports three integration scenarious:
+
+- **Pure In-App** bidding without Primary Ad Server
+- **With Google** Ad Manger as a primary Ad Server
+- **With MoPub** as a primary ad Server
+
+### Pure In-App bidding
+
+![How Prebid Mobile Works]({{site.baseurl}}/assets/images/prebid-mobile/rendering/prebid-in-app-bidding-overview-pure-prebid.png){: .pb-lg-img :}
+
+1. Prebid SDK sends the bid request to the Prebid Server.
+1. Prebid Server runs the header bidding auction among preconfigured demand partners.
+1. Prebid Server responses with the winning bid.
+1. Prebid SDK renders the creative from recieved bid.
```

Comment:
Prebid SDK renders an ad.

We should figure out how we support rendering with different bid types, such as deals or deal types like Programmatic Guaranteed. Lets just say it renders an ad for now since we have not yet worked out how ad types would work.